packages/asc-ui/src/utils/hooks/useTrappedFocus.ts

```diff
@@ -2,7 +2,7 @@ import { KeyboardEvent, RefObject } from 'react'
 import { KeyboardKeys } from '../../types'
 import ownerDocument from '../ownerDocument'
 
-const FocusableElements = [
+export const FocusableElements = [
```

Comment:
Can you rename this one to use camel case?
```suggestion
export const focusableElements = [
```